check

```diff
@@ -0,0 +1 @@
+python -m pytest -sq tests/environment_test.py
```

Comment:
Hm, my comment about pytest being a dev dependency typically was also meant as removing it from check the check command. I think I wasn't very clear in my explanation. My point was that any testing library is usually just used for testing and not for actually running the software itself. Because of that it's usually a dev dependency because the people running the software itself don't need it. I would say the check command is something that is not for devs but rather for normal users. So it just gave me a bit of an ick that normal users would need to install a testing library to run it. Especially since the check command isn't even the actual software itself. But, as I mentioned, it's not a big deal. But with the current version it's kind of the worst of both worlds because it's now listed as a dev dependency but in order to run the check command it is still needed. So if a normal user would want to run the check command they would need to install the dev dependencies. I'm fine with each approach (removing pytest from check as a dependency or just making pytest not a dev dependency anymore) but it should be made consistent again. Sorry that my previous comment was confusing, I hope this makes sense.

Reply:
Ok, yea I see, sorry I misunderstood the problem here. If we make this a script and not a test it makes more sense, see [014b99f](https://github.com/asmap/kartograf/pull/26/commits/014b99f574f8c7629457079e24dbdcb76d5cc6aa). This way the user only requires python, and everything else gets pretty printed whether its in the path or not. lmk if this works.